worlds/witness/player_logic.py

```diff
@@ -1024,4 +1089,7 @@ def make_event_panel_lists(self) -> None:
             entity_name = entity_obj["checkName"]
             self.EVENT_ITEM_PAIRS[entity_name + " (Panel Hunt)"] = ("+1 Panel Hunt", entity_hex)
 
+        for region_name, easter_egg_count in self.AVAILABLE_EASTER_EGGS_PER_REGION.items():
+            self.EVENT_ITEM_PAIRS[f"{region_name} Easter Eggs"] = (f"+{easter_egg_count} Easter Eggs", region_name)
```

Comment:
```suggestion
        for region_name, easter_egg_count in self.AVAILABLE_EASTER_EGGS_PER_REGION.items():
            plural = "s" if easter_egg_count != 1 else ""
            self.EVENT_ITEM_PAIRS[f"{region_name} Easter Egg{plural}"] = (f"+{easter_egg_count} Easter Egg{plural}", region_name)
```

Reply:
Oh lol, I fixed the things that would be broken by this, and then didn't actually do it.